src/TagsGroup.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { CSSProperties, forwardRef, memo } from "react";
+import { assert, Equals } from "tsafe";
+import { symToStr } from "tsafe/symToStr";
+import Tag, { TagProps } from "./Tag";
+import { useAnalyticsId } from "./tools/useAnalyticsId";
+import { cx } from "./tools/cx";
+import { fr } from "./fr";
+
+export type TagsGroupProps = TagsGroupProps.Common;
+
+//https://main--ds-gouv.netlify.app/example/component/tag/#:~:text=Groupe%20de%20tags
+export namespace TagsGroupProps {
+    export type Common = {
+        id?: string;
+        className?: string;
+        style?: CSSProperties;
+        smallTags?: TagProps["small"];
```

Comment:
Can you add a comment above the property to document the default value, like this? `/** default: false */`